src/Process/3dTilesProcessing.js

```diff
@@ -88,6 +132,20 @@ export function pre3dTilesUpdate(context, layer) {
     // const HFOV = 2.0 * Math.atan(Math.tan(radAngle * 0.5) / context.camera.ratio);
     const HYFOV = 2.0 * Math.atan(Math.tan(radAngle * 0.5) * hypotenuse / context.camera.width);
     context.camera.preSSE = hypotenuse * (2.0 * Math.tan(HYFOV * 0.5));
+
+    // once in a while, garbage collect
+    if (Math.random() > 0.98) {
+        // Browse
+        const now = Date.now();
+
+        for (const elt of layer._cleanableTiles) {
+            if ((now - elt.cleanableSince) > (layer.cleanupDelay || 1000)) {
```

Comment:
Can we make this option more discoverable? (document it... somewhere?)

Reply:
So I guess the question is: where do we document options that are specific for one type of layers?

I guess in this case, it should be it the `Processing` module. Maybe we can document what layer options this `Processing` file use on top of it?

Reply:
This is an important comment, but IMHO we should make a specific issue/PR to properly document all layers options.

Reply:
We usually initialize these kind of options in preprocessDataLayer, which could be a first step in making this parameter more discoverable.

Reply:
Followed your suggestion and moved this to `preprocessDataLayer`